lib/vre/ch31_form.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require_relative 'service'
+require 'sentry_logging'
+
+module VRE
+  class Ch31Form < Service
+    include SentryLogging
+    class VRECH31Error < StandardError
+    end
+
+    def initialize(user, claim)
+      @user = user
+      @claim = claim
+      @parsed_form = @claim.parsed_form
+    end
+
+    def submit
+      send_to_vre(format_payload_for_vre, VRECH31Error)
```

Comment:
The `send_to_vre` method could use keyword args to help clarify what args are required. Also, it could be more elegant to include an ErrorMiddleware extension to the Faraday configuration that handles custom error classes, but not necessary for this iteration -- keep it in mind as you add more errors to this module.